addons/payment_authorize/static/src/js/payment_form.js

```diff
@@ -21,11 +21,16 @@ odoo.define('payment_authorize.payment_form', require => {
          */
         _getInlineFormInputs: function (providerId) {
             if (this.authorizeInfo.payment_method_type === "credit_card") {
+               let form = document.getElementById(`o_authorize_${this.formType}_${providerId}`);
+    
+               if (!form){
+                    form = document.getElementById(`o_authorize_form_${providerId}`);
+               }
```

Comment:
This is just a suggestion, I like your version too, I won't enforce this. (mind the line limit if you choose it though)

```suggestion
               let form = document.getElementById(`o_authorize_${this.formType}_${providerId}`) || document.getElementById(`o_authorize_form_${providerId}`);
```